pkg/sfu/buffer/buffer.go

```diff
@@ -18,22 +18,20 @@ import (
 	"go.uber.org/atomic"
 )
 
-const (
-	ReportDelta = 1e9
-)
-
 type pendingPacket struct {
 	arrivalTime int64
 	packet      []byte
 }
 
 type ExtPacket struct {
-	Head      bool
-	Arrival   int64
-	Packet    *rtp.Packet
-	Payload   interface{}
-	KeyFrame  bool
-	RawPacket []byte
+	Head          bool
+	Arrival       int64
+	Packet        *rtp.Packet
+	Payload       interface{}
+	KeyFrame      bool
+	RawPacket     []byte
+	SpatialLayer  int32
```

Comment:
seems that putting these in ExtPacket will make it easier for us to support AV1/VP9 and the likes?

Reply:
Yes, that is the intention.